Translated/UI/ui_mainmode.csv

```diff
@@ -3502,9 +3502,9 @@ QuestName#0,"""クエスト名"""
 QuestTarget#0,"""クエスト目標"""
 Trial#0,"""Eトライアル「$(0)」"""
 Rank#0,"""($(0))"""
-Count#1,"""（$(1)：+$(0)）"""
-NowCount#0,"""（現在：+$(0)）"""
-FloorCount#0,"""(到達ステージ$(0)）"""
+Count#1,"""($(1)：+$(0))"""
+NowCount#0,"""(現在：+$(0))"""
+FloorCount#0,"""(到達ステージ$(0))"""
```

Comment:
Please don't touch the punctuation on lines that aren't otherwise translated.

Reply:
resolved